CMakeLists.txt

```diff
@@ -108,6 +108,14 @@ if (#[[${CMAKE_VERSION} VERSION_LESS "3.18" AND]] ${CMAKE_SYSTEM_NAME} MATCHES "
     string(REPLACE "-framework;" "-framework " libs "${libs}")
     message(STATUS "PkgConfig::ExternalModules::INTERFACE_LINK_LIBRARIES fixed: ${libs}")
     set_target_properties(PkgConfig::ExternalModules PROPERTIES INTERFACE_LINK_LIBRARIES "${libs}")
+
+
+    if (${CMAKE_VERSION} VERSION_GREATER_EQUAL "3.13")
```

Comment:
This looks good so far, but it is fixed since [3.18](https://gitlab.kitware.com/cmake/cmake/-/merge_requests/3814), so either exclude newer versions or raise the minimum required for macOSX.

Reply:
It is not fixed. I am using CMake 3.22.